source/mainnet/net/resources/terms-and-conditions-mw-gen2.rst

```diff
@@ -0,0 +1,161 @@
+.. _terms-and-conditions-mw-gen2:
+
+=================================================
+Terms and Conditions for Concordium Mobile Wallet
```

Comment:
Hmm. The title is not consistent with the name of the app. I guess this is something we have to update throughout the T&C once the naming is final

Reply:
Mikael told me specifically that the names in the T&C were not updated and would be updated later. This was part of the same message where he told me the updated names. I am not sure why the updated names are not used in the T&C. That would be a question for Mikael.